exercises/resistor-color/canonical-data.json

```diff
@@ -0,0 +1,41 @@
+{
+  "exercise": "resistor-color",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Color codes",
+      "cases": [
+        {
+          "description": "Black",
+          "property": "colorCode",
+          "input": {
+            "color": "black"
+          },
+          "expected": "0"
```

Comment:
Are strings the intended output or should it be integer type?

Reply:
That's a great point! It should be integers. Agree, @ErikSchierboom ?

Reply:
It should definitely have been integers. Fixed!